docs: add breaking changes guide regarding echart changes (#1815)
siemens/ix: BREAKING_CHANGES/v3.md

#### ix-typography

- The `color` attribute has been **replaced by** `textColor`.

## `@siemens/ix-echarts`

### Theme names changed

- From `brand-light` to `theme-brand-light`
- From `brand-dark` to `theme-brand-dark`
- From `classic-light` to `theme-classic-light`
- From `classic-dark` to `theme-classic-dark`

> e.g
> 
> From:
> 
> ```javascript
> import { convertThemeName, registerTheme } from '@siemens/ix-echarts';
> 
> const myEChart = echarts.init(element, convertThemeName(theme));
> ```
> 
> To:
> 
> ```javascript
> import { registerTheme } from '@siemens/ix-echarts';
> 
> const myEChart = echarts.init(element, theme);
> ```
> 
> ### Utility function `convertThemeName` is removed from `@siemens/ix-echarts`
> 
> Utility function `convertThemeName` has be removed, because not needed anymore after aligning echarts names with ix css theme names.

Reviewer: ```suggestion
### Utility function `convertThemeName` war removed from `@siemens/ix-echarts`

The `convertThemeName` utility function has been removed, because it was not needed anymore after aligning the echarts theme names with iX CSS theme names.

From:

```javascript
import { convertThemeName, registerTheme } from '@siemens/ix-echarts';

const myEChart = echarts.init(element, convertThemeName(theme));
```

To:

```javascript
import { registerTheme } from '@siemens/ix-echarts';

const myEChart = echarts.init(element, theme);
```
```
